lib/index.js

```diff
@@ -10,10 +10,13 @@ var oauth2_client;
  *  @returns {Object} oauth2_client - the Google OAuth2 client
  */
 function create_oauth2_client () {
+  var google_client_id = (OPTIONS.google_client_id) ? OPTIONS.google_client_id : process.env.GOOGLE_CLIENT_ID;
```

Comment:
is this to allow the person to pass in the `GOOGLE_CLIENT_ID` on the `OPTIONS` object? I don't see where this is supported in the plugin code.